c-sharp-chat/PubnubChatApi/PubnubChatApi/Entities/User.cs

```diff
@@ -179,6 +179,23 @@ public string DataType
             }
         }
 
+        public bool Active
+        {
+            get
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        public string LastActiveTimeStamp
+        {
+            get
+            {
+                var buffer = new StringBuilder(64);
+                throw new NotImplementedException();
+            }
+        }
+
```

Comment:
Maybe we should comment it out for now? What do you think?

Reply:
Oh right, commented them out